docs/installation/dns_setup.md

```diff
@@ -1,157 +1,201 @@
+Ingress controllers are such services (for example, Traefik) so should have an externally accessible IP address.
+This is needed for any Ingress resource to work.
+
+You can find the load balancer IP address of any service using `kubectl`. For example:
 
 ```
 kubectl get svc -n kube-system traefik -o jsonpath={@.status.loadBalancer.ingress}
 ```
+<!-- TODO: I would like a sample output from the previous command showing the IP address field.-->
 
-Depending on the load balancer, the result of this command may have different fields populated, most usually `ip`.
-Keep this information available for the next step where you will configure your DNS.
+Depending on the load balancer, the result of this command may have different fields populated.
+We are interested in the `ip` field.
+You use the IP field in the next step to configure your DNS.
 
-## Configure your DNS
+## Configure your DNS {#config-dns}
 
-Given you own the domain **"example.com"**, you would configure a subdomain e.g. **"test.example.com"** for Epinio.
-Now you will have to configure your DNS in a way, that any request towards "test.example.com" will resolve to the address found in the previous step.
+Assuming you own the domain `example.com`, you would configure a subdomain, for example, `test.example.com` for Epinio.
+You can now configure your DNS so that any request for `test.example.com`
+resolves to the address you obtained in the previous step.
 
-- test.example.com => "INGRESS-IP"
-- \*.test.example.com => "INGRESS-IP"
+<!--TODO: We have described what needs to be configured in DNS, but not actually shown how to do it.
+Do we need to? Or am I missing the point.-->
 
-***
+- `test.example.com => "INGRESS-IP"`
+- `*.test.example.com => "INGRESS-IP"`
 
-###### The "wildcard" entry is what makes the automatic routes for applications to work in Epinio. Don't skip it.
+:::tip
+The setup of a wildcard entry is important and allows automatic routing for applications to work in Epinio.
+:::
 
-***
+We have some DNS configuration examples in the next section
 
-Find [DNS Configuration Examples](#dns-configuration-examples) below.
+## DNS configuration examples
 
-## DNS Configuration Examples
+### Amazon Route53 & AWS EKS
 
-### AWS EKS and Route53
+We will use the [Amazon Route53](https://aws.amazon.com/route53/) to create a wildcard domain within an existing "Hosted zone", like `example.com`.
 
-As an example we will use the [AWS Service Route53](https://aws.amazon.com/route53/) to create a wildcard domain within one of your existing "Hosted zones", e.g. **example.com**.
-
-Given Epinio ingress installation provided you with the following hostname:
+Suppose an Epinio ingress installation has provided you with the following hostname:
+<!--TODO: Is the hostname below an actual format of an Amazon AWS hostname.
+If so that's fine, if not I think I will shorten it for a little more clarity.-->
```

Comment:
At first glance that looks correct. @mmartin24 @thehejik might know better.

Reply:
Ok, I may exaplain and use something shorter anyway. Unless @mmartin24 or @thehejik have objections.

Reply:
Looks good to me, no objections, the hostname format matches with what we get from EKS.